apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/ShowTokenActions/ClearQuantities.tsx

```diff
@@ -18,5 +18,5 @@ export const ClearQuantities = () => {
 }
 
 const styles = StyleSheet.create({
-  text: {color: COLORS.SHELLEY_BLUE, fontWeight: '500', textTransform: 'uppercase'},
+  text: {color: COLORS.SHELLEY_BLUE, fontWeight: '600', textTransform: 'uppercase'},
```

Comment:
600 is bold right?

I suggest to use fontWeight: 'bold' because there is a weird issue on iOS  or android (dont 
remember)
```suggestion
  text: {color: COLORS.SHELLEY_BLUE, fontWeight: 'bold', textTransform: 'uppercase'},
```

Reply:
no, not is bold